configuration/config-txt/boot.md

```diff
@@ -113,7 +113,7 @@ On the Raspberry Pi 4B, if this value is set to `0` then the interrupts will be
 
 ## force_eeprom_read
 
-Set this option to `0` to prevent the firmware from trying to read an I2C HAT EEPROM (connected to pins ID_SD & ID_SC) at powerup.
+Set this option to `0` to prevent the firmware from trying to read an I2C HAT EEPROM (connected to pins ID_SD & ID_SC) at powerup. See also `disable_poe_fan`.
```

Comment:
You probably want this to be a hyperlink, in which case you would need to add an anchor before the heading for `disable_poe_fan` in misc.md. For example:

`<a name="disable-poe-fan"></a>
## disable_poe_fan`

You can then link to using Markdown syntax, e.g.

See also [`disable_poe_fan`](misc.md)

Reply:
Thanks for the suggestion. I considered this, but could quickly only see examples of links *within* a page, not across pages. However, looking more closely, it seems links across pages are also used, so I followed those examples and amended my commit with a link added. Seems to work in the github rendering of the markdown, so I think it's good now :-)